README.md

```diff
@@ -36,7 +36,10 @@ This middleware is implemented using the [`http-proxy`](https://www.npmjs.com/pa
 - You can access the `http-proxy` instance using the `onProxyInit` option. See the example below.
   
   ```ts
-  const handleProxyInit = (proxy: httpProxy) => {
+  import type { NextApiRequest, NextApiResponse } from "next";
+  import httpProxyMiddleware from "next-http-proxy-middleware";
+
+  const handleProxyInit = (proxy: import("http-proxy")) => {
```

Comment:
Hi @bever1137 Thanks for your PR! 😀
Is there a reason why you did http-proxy import inline rather than top of the file?

Reply:
Since the type isn't reused, I didn't see the advantage of adding it to the module scope. I can see how it is clearer to `import type...` at the top instead. I'll make the change.